docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -98,21 +98,25 @@ In order to make sure this works for either local or remote clients, we have to
 Docker network and expose the required ports correctly.
 
 There are two ways to launch Alluxio Docker containers on the Docker host:
- + Use [host network](https://docs.docker.com/network/host/) to
-  share ip-address and networking namespace between the container and the Docker host.
- + Use [user-defined bridge network](https://docs.docker.com/network/bridge/) to
-  allow containers connected to communicate, while providing isolation from containers which are
-  not connected to that bridge network.
++ A: Use [host network](https://docs.docker.com/network/host/)
++ B: Use [user-defined bridge network](https://docs.docker.com/network/bridge/)
+
+Host network shares ip-address and networking namespace between the container and the Docker host.
+User-defined bridge network allows containers connected to communicate,
+while providing isolation from containers not connected to that bridge network.
+For the purpose of test, using host network is sufficient.
```

Comment:
It is recommended to use host network, option A, for testing.

Reply:
ack